bot/slack/commands/ip_whois.py

```diff
@@ -0,0 +1,31 @@
+import netaddr
+from ipwhois import IPWhois
+
+
+def is_ip(ip):
```

Comment:
We should prefer to move shared code to a module where it can be imported in both cases. Unnecessary duplication will get us back to a point where we have modified duplicates littered everywhere and make it difficult to refactor.

Reply:
I moved this into mozdef_util at https://github.com/mozilla/MozDef/pull/1018/commits/5f67c2d9eb99ab1a74e8420b6eb08334e9cbf241 . The build will fail because we had to bump the version of mozdef-util in requirements.txt, but I haven't built and pushed a new version of mozdef-util yet, so if you could take a look at the previously mentioned changes and give a thumbs up or down, and I'll then push a new version of mozdef-util to pypi, which should cause the build to then start passing.